### DISCUSSION TITLE
Save as audio default template

### DISCUSSION THREAD

**Discussion by thepaleguy** [JunkFood02/Seal#359] [Q&A]
what is the default template code for saving as audio?

**Comment by BobbyESP** [comment]
You can simply click on Save as audio on the dialog that is opened before the download starts and the video will be downloaded just as an audio.

**Comment by JunkFood02** [comment]
[source code](https://github.com/JunkFood02/Seal/blob/main/app/src/main/java/com/junkfood/seal/util/DownloadUtil.kt)
